### ISSUE TITLE
app.Stop() and app.Suspend() require an additional keypress before clearing screen

### ISSUE THREAD

**Issue by lukegalea** [rivo/tview#597]
Both when using app.Stop() to exit a tview application, or when using app.Suspend(), I'm finding that a second keypress is required before the user actually "sees" the app stop.

in the trivial example below, a single press of escape should exit the program... but instead, the first press of escape doesn't clear the screen nor exit the program.. but a second keypress of any kind (enter, a second escape.. whatever) does trigger the program closing and the screen clearing.

I've tried seeding the program w/ debug output (fmt.Println) to confirm that app.Stop is being called.. and it is.. it just doesn't have the desired effect.

```
package main

import (
	"github.com/gdamore/tcell/v2"
	"github.com/rivo/tview"
)

func main() {
	app := tview.NewApplication()

	app.SetInputCapture(func(event *tcell.EventKey) *tcell.EventKey {
		if event.Key() == tcell.KeyEsc {
			app.Stop()
		}
		return event
	})

	app.
		SetRoot(tview.NewTextView().SetText("Hello World"), true).
		Run()
}
```

**Comment by lukegalea** [comment]
A colleague has confirmed for me that this is not a problem on OSX. A single press of escape clears the screen and exits the process.,

I've confirmed it to be a problem on multiple linux distros, though.

**Comment by mdom** [comment]
This seems to be the same issue mentioned in gdamore/tcell#452. it works as intended when i'm building with tcell 2.2.0.

**Comment by mdom** [comment]
This is fixed in gdamore/tcell@8f925d8 and works fine on my machine. Maybe it would be good to update the dependency on tcell to release 2.3.1?

**Comment by D54** [comment]
I can confirm that the current version works perfectly with gdamore/tcell v2.3.3.
c723ed0 solved this issue as well, so it can be closed.
